lib/Code/TidyAll.pm

```diff
@@ -252,6 +252,10 @@ sub new_from_conf_file {
     die "no such file '$conf_file'" unless $conf_file->is_file;
     my $conf_params = $class->_read_conf_file($conf_file);
     my $main_params = delete( $conf_params->{'_'} ) || {};
+    my $global_ignores = delete $main_params->{'ignore'} || [];
+    for my $plugin_name (keys %{ $conf_params }) {
+        push @{ $conf_params->{$plugin_name}->{'ignore'} || []}, @$global_ignores;
```

Comment:
I don't think altering the plugin config like this is the way to make this work. I know it's the easiest approach, but it doesn't seem right to me.

I think it'd be better to add an attribute to the `Code::TidyAll` object called `global_ignores` and then use that in the `find_matched_files` method.